### PR TITLE
Create churchmetrics.json

### PR DESCRIPTION
**Pull request by bmcclenahan**

### CODE REVIEW

File: organizations/churchmetrics.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+  "id": "CHURCHMETRICS",
+  "name": "Church Metrics",
```
Comment (AnuFeb) [inline organizations/churchmetrics.json:3]: Use lowercase characters and underscores for spaces. The filename should match the Id property of the source definition

**Review by AnuFeb — APPROVED** [review]
LGTM, Thanks for adding!

**Comment by AnuFeb** [comment]
Could you please update the branch and I will merge the change.

**Comment by bmcclenahan** [comment]
branch updated

**Comment by AnuFeb** [comment]
Please update the branch and I will merge the change.

**Comment by bmcclenahan** [comment]
branch updated again

**Pull request merged**